datastore/google/cloud/datastore/client.py

```diff
@@ -302,15 +304,27 @@ def get(self, key, missing=None, deferred=None, transaction=None):
         :param transaction: (Optional) Transaction to use for read consistency.
                             If not passed, uses current transaction, if set.
 
+        :type eventual: bool
+        :param eventual: (Optional) Defaults to strongly consistent (False).
+                                    Setting True will use eventual consistency,
```

Comment:
"Setting" should align with "(Optional) ..."

Reply:
@chemelnucfin In the future, please address all review comments (like this one).